SimPEG/utils/io_utils/io_utils_electromagnetics.py

```diff
@@ -44,51 +217,233 @@ def read_dcip3d_ubc(file_name, data_type):
+        "simple"
+    ], "Parameter 'format_type' must be one of {'general', 'surface', 'simple'}"
+    
+    # Prevent circular import
+    from ...electromagnetics.static import resistivity as dc
+    from ...data import Data
 
-    return _read_dcip_3d_or_octree_ubc(file_name, data_type, "dcip3d")
-
-
-def read_dcipoctree_ubc(file_name, data_type):
-    """
-    Read UBC DCIPoctree formatted survey, predicted or observed data files.
-
-    This function loads 3D DC or IP data formatted for the UBC DCIPoctree
-    coding package and outputs a SimPEG data object. The function requires
-    the user to define whether the data are DC resistivity or IP.
-
-    Parameters
-    ----------
-    file_name : str
-        The file path to the data file
-    data_type: str {'volt', 'apparent_chargeability'}
-        Defining the input data type:
-
-        - 'volt': DC resistivity data as voltages
-        - 'apparent_chargeability': IP data as apparent chargeabilities
+    # Load file
+    obsfile = np.genfromtxt(file_name, delimiter="\n", dtype=np.str, comments="!")
 
-    Returns
-    -------
-    data_object
-        A SimPEG.data.Data object containing:
+    # Find starting data
+    start_index = 0
+    if obsfile[0] == 'COMMON_CURRENT':
+        start_index = 2
+    if data_type != "volt":
+        start_index = start_index + 1
+    
+    obsfile = obsfile[start_index:]
 
-        - The survey
-        - Observed/predicted data (if present in the data file)
-        - Uncertainties (if present in the data file). Note that predicted DC data
-        files contain the apparent resistivities, which are loaded into SimPEG and
-        defined as uncertainties.
+    # Since SimPEG defines secondary potential from IP as voltage,
+    # we must use this type when defining the receivers.
+    if data_type.lower() == "secondary_potential":
+        data_type = "volt"
+    
+    # Pre-allocate
+    source_list = []
+    receiver_list = []
+    d = []
+    wd = []
 
-    """
+    # Flag for z value provided
+    is_surface = False
+    is_pole_tx = False
+    is_pole_rx = False
 
-    # Unused for now but it will be when we manage IP types better.
-    assert data_type.lower() in [
-        "volt",
-        "apparent_chargeability",
-    ], "Parameter 'data_type' must be one of {'volt', 'apparent_chargeability'}"
+    if format_type.lower() == 'simple':
+        
+        # Load numeric data into an array
+        if data_type.lower() == 'volt':
+            data_array = np.loadtxt(file_name, comments="!", skiprows=start_index)
+        else:
+            data_array = np.loadtxt(file_name, comments={"!", "IPTYPE"}, skiprows=start_index)
+            
+        n_rows = np.shape(data_array)[0]
+        
+        if np.shape(data_array)[1] > 5:
+            wd = data_array[:, 5]
+        
+        if np.shape(data_array)[1] > 4:
+            d = data_array[:, 4]
+        
+        # Get ABMN electrode locations
+        dummy_elevation = 9999
```

Comment:
again, probably set this as `np.inf` or `np.nan` so the simulations will actually fail and not necessarily run while the user ignores the warning.

Reply:
The application of topography to the data is done in the simulations. So if someone goes through the tutorials properly, they will end up applying the topography.